File: banking-web/src/app/shared/vo/account-vo.ts
import { ClientVo } from "./client-vo";


export class AccountVo {
    accountId!: number;

    accountNumber!: string;

    accountType!: string;

    accountBeginBalance!: number;

    accountStatus!: boolean;

    client!: ClientVo;

    movementEntities!: any[];
}

File: banking-root/src/main/java/com/devsu/banking/vo/AccountVo.java
package com.devsu.banking.vo;

import com.devsu.banking.client.entity.ClientEntity;
import com.devsu.banking.client.entity.MovementEntity;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class AccountVo {

    private Integer accountId;

    private String accountNumber;

    private String accountType;

    private Double accountBeginBalance;

    private Boolean accountStatus;

    private ClientEntity client;

    private List<MovementEntity> movementEntities;

}
